README.md

```diff
@@ -7,12 +7,11 @@
 [![Slack](https://img.shields.io/badge/slack-@kyma--community-yellow.svg)](http://slack.kyma-project.io)
 [![Twitter](https://img.shields.io/badge/twitter-@kymaproject-blue.svg)](https://twitter.com/kymaproject)
 
-
 ## Overview
 
-Kyma allows you to connect applications and third-party services in a cloud-native environment. Use it to create extensions for the existing systems, regardless of the language they are written in. Customize extensions with minimum effort and time devoted to learning their configuration details.
+**Kyma** `/kee-ma/` is a platform for extending applications with microservices and [serverless](https://kyma-project.io/docs/components/serverless/) functions. It provides CLI and UI through which you can connect your application to a Kubernetes cluster and expose the application's API or Events securely thanks to the in-built [Application Connector](https://kyma-project.io/docs/components/application-connector/#overview-overview). You can then implement the business logic you require by creating microservices or functions, and triggering them to react to particular Events or calls to your application's API. To limit the time spent on coding, use the in-built cloud services from the [Service Catalog](https://kyma-project.io/docs/components/service-catalog/), exposed by [Service Brokers](https://kyma-project.io/docs/components/service-catalog/#service-brokers-service-brokers) from such cloud providers as GCP, Azure, and AWS.
 
-Go to the [Kyma project website](https://kyma-project.io/) to learn more about the product, its features, and components.
+Go to the [Kyma project website](https://kyma-project.io/) to learn more about our project, its features, and components.
```

Comment:
if we will have here what is on the website just change this text to link to docs only, or remove at all and have the links we have on a website (the ones we discussed today, to all features and to components)

Reply:
as above